services/src/northbound/src/main/java/org/openkilda/northbound/utils/RequestCorrelationFilter.java

```diff
@@ -50,7 +50,7 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         String correlationId = request.getHeader(CORRELATION_ID);
         if (StringUtils.isBlank(correlationId)) {
             correlationId = UUID.randomUUID().toString();
-            LOGGER.warn("CorrelationId was not sent, generated one: {}", correlationId);
+            LOGGER.debug("CorrelationId was not sent, generated one: {}", correlationId);
```

Comment:
why do you lower debug level?

Reply:
I'm not sure that it is really warning and we need such kind of information. Currently there are too many such messages that make no sense and we might missed really some warning messages.